geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreColumnGroupsTest.scala

```diff
@@ -101,7 +101,7 @@ class AccumuloDataStoreColumnGroupsTest extends Specification with TestWithDataS
     ("bbox(geom,45.4,54.9,45.8,55.1) and height < 67", 4 to 6),
     ("age >= 50 and height < 67", 5 to 6),
     ("bbox(geom,45.4,54.9,45.8,55.1) AND name IN ('name5', 'name6') and height < 67", 5 to 6),
-    ("bbox(geom,45.4,54.9,45.8,55.1) AND age >= 50 and height < 67", 5 to 6),
+    ("bbox(geom,45.41,54.9,45.8,55.1) AND age IS NOT NULL and height < 67", 5 to 6),
```

Comment:
Why the change in the filter here?

Reply:
with the change to heuristic query planning, the filter was hitting a different index, so i tweaked it to keep it against the z3